### PR TITLE
Zpi 110: Better listing and create fix 

### PR DESCRIPTION
**Pull request by Ninja00Shadow**
I’ve added filtering for resources (and releases) by namespace and name. I also added a namespace input in the create, but I’m not happy with it. I’d appreciate suggestions.

### CODE REVIEW

**Review by vgoriukhina — COMMENTED** [review]
I think that in the "Create" section, where it's possible to enter a namespace, it would be better to add a "Namespace" label next to the field and place "default" as placeholder text inside the input. This way, it will be clear from the beginning that "default" is assigned and that it pertains to the namespace.

**Comment by Ninja00Shadow** [comment]
> I think that in the "Create" section, where it's possible to enter a namespace, it would be better to add a "Namespace" label next to the field and place "default" as placeholder text inside the input. This way, it will be clear from the beginning that "default" is assigned and that it pertains to the namespace.

Changed placeholder to default. This should be enough.

**Comment by Nightbaka** [comment]
> > I think that in the "Create" section, where it's possible to enter a namespace, it would be better to add a "Namespace" label next to the field and place "default" as placeholder text inside the input. This way, it will be clear from the beginning that "default" is assigned and that it pertains to the namespace.
> 
> Changed placeholder to default. This should be enough.

Hmmm what about the namespace-less resources case

File: charts/kam/Chart.yaml [hunk 1]
```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.10
+version: 1.1.11
```
Comment (Nightbaka) [inline charts/kam/Chart.yaml:18]:
```suggestion
version: 1.1.12
```

File: charts/kam/Chart.yaml [hunk 2]
```diff
@@ -15,10 +15,10 @@ type: application
...
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.1.10"
+appVersion: "1.1.11"
```
Comment (Nightbaka) [inline charts/kam/Chart.yaml:24]:
```suggestion
appVersion: "1.1.12"
```

File: frontend/src/components/Menu/Menu.tsx [hunk 3]
```diff
@@ -25,8 +25,8 @@ const LeftMenu: React.FC = () => {
                     children: generateMenuItems(item.children),
                 };
             }
-            const disabled = item.label !== 'Roles' ? (permissions !== null && !hasAnyPermissionInAnyNamespace(permissions, item.resourcelabel)) :
-                (permissions !== null && !hasPermission(permissions, ROLEMAP_NAMESPACE, 'ConfigMap', 'r'));
+            const disabled = item.label !== 'Roles' ? (permissions === null || !hasAnyPermissionInAnyNamespace(permissions, item.resourcelabel)) :
+                (permissions === null || !hasPermission(permissions, ROLEMAP_NAMESPACE, 'ConfigMap', 'r'));
```
[the remark this answers is not in the document]
Comment (Ninja00Shadow) [reply frontend/src/components/Menu/Menu.tsx:29]: Jeżeli permissions === null to by przeszło

[the remark this answers is not in the document]
Comment (Nightbaka) [reply frontend/src/components/Menu/Menu.tsx:29]: No i o to chodzi

File: frontend/src/components/Table/Tab.module.css [hunk 4]
```diff
@@ -16,5 +23,20 @@
     background-color: #bae7ff !important;
 }
 
+.controlSection {
+    display: flex;
```
Comment (samuelus) [inline frontend/src/components/Table/Tab.module.css:27]: a
a
a
a
a
a

a
a
a
a
a
a
a
a
a
a
a
a
a
a
a
a

a
a
a

**Review by vgoriukhina — APPROVED** [review]

**Pull request merged**